frontend/packages/dev-console/src/components/helm/HelmReleaseList.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { Table, TextFilter } from '@console/internal/components/factory';
+import { SortByDirection } from '@patternfly/react-table';
+import { CheckBoxes } from '@console/internal/components/row-filter';
+import { FirehoseResult, getQueryArgument } from '@console/internal/components/utils';
+import { HelmRelease, HelmFilterType } from './helm-types';
+import { helmRowFilters, getFilteredItems, useDeepCompareMemoize } from './helm-utils';
+import HelmReleaseHeader from './HelmReleaseHeader';
+import HelmReleaseRow from './HelmReleaseRow';
+
+interface HelmReleaseListProps {
+  namespace: string;
+  secrets?: FirehoseResult;
+}
+
+const HelmReleaseList: React.FC<HelmReleaseListProps> = ({ namespace, secrets }) => {
+  const [releases, setReleases] = React.useState([]);
+  const [filteredReleases, setFilteredReleases] = React.useState([]);
+  const [fetched, setFetched] = React.useState(false);
+
+  const memoizedSecrets = useDeepCompareMemoize(_.get(secrets, 'data'));
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const activeFilters =
+      getQueryArgument('rowFilter-helm-release-status') &&
+      getQueryArgument('rowFilter-helm-release-status').split(',');
```

Comment:
You should only call getQueryArgument once and store the value in a local variable.

Reply:
Making it a local variable outside of `useEffect` will result in unnecesarry calls to `useEffect` and API whenever there is a change in filter.

Reply:
I don't mean in a variable outside the effect. I meant this:
```suggestion
      const queryArgument = getQueryArgument('rowFilter-helm-release-status');
      const activeFilters = queryArgument &&  queryArgument.split(',');
```

Reply:
Done.